pegasus/sites.v3/code.org/public/private/poste/index.haml

```diff
@@ -83,7 +83,8 @@
     Showing most recently sent mail. Click to view all mail.
 
 :ruby
-  rows = DB.fetch("
+  POSTE_REPORTING_DB_READONLY = sequel_connect(CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader)
```

Comment:
Yes, I'm pretty sure it is and that's where `DB` comes from, but I'm not positive. Pegasus dependencies are a bit hard to trace. Is there an existing `PEGASUS_REPORTING_DB_READONLY`?